modules/ob-creating-buildah-build-in-a-network-restricted-environment.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+//* builds/work-with-builds.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id='ob-creating-buildah-build-in-a-network-restricted-environment_{context}']
+= Creating buildah build in a network restricted environment
```

Comment:
Should this be _restricted network_? I see  [network restricted environment is used elsewhere in the Builds docs](https://docs.openshift.com/builds/1.2/configuring/using-builds-in-a-network-restricted-environment.html). 
If network restricted environment and restricted network are not the same thing, we should at least go with _network-restricted environment_.
Can be done in a separate PR. Not worth holding up the merge.